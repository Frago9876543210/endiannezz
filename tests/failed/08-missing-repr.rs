use endiannezz::Io;

#[derive(Io)]
#[endian(big)]
enum Foo {
    Bar,
    Baz,
}

fn main() {}
