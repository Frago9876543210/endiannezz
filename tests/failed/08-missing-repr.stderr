error: Enums must declare #[repr]
 --> $DIR/08-missing-repr.rs:4:1
  |
4 | / #[endian(big)]
5 | | enum Foo {
6 | |     Bar,
7 | |     Baz,
8 | | }
  | |_^
